diff --git a/settings.gradle.kts b/settings.gradle.kts
index 092f6d7c2..93ea9feab 100644
--- a/settings.gradle.kts
+++ b/settings.gradle.kts
@@ -31,12 +31,6 @@ val desktopOnly =
     providers.gradleProperty("desktop.only").orNull?.toBoolean() == true ||
         System.getenv("DESKTOP_ONLY")?.toBoolean() == true
 
-plugins {
-    id("org.gradle.toolchains.foojay-resolver") version "1.0.0"
-    id("com.gradle.develocity") version "4.4.1"
-    id("com.gradle.common-custom-user-data-gradle-plugin") version "2.6.0"
-}
-
 @Suppress("UnstableApiUsage")
 dependencyResolutionManagement {
     repositoriesMode = RepositoriesMode.FAIL_ON_PROJECT_REPOS
